.github/workflows/publish-cloud.yml

```diff
@@ -5,123 +5,26 @@ on:
-    needs: bump-version
+  publish-remote-repo:
     runs-on: ubuntu-20.04
     steps:
-      - name: Checkout branch
-        uses: actions/checkout@v3
-      - name: Install Node.js 14.x
-        uses: actions/setup-node@v2
-        with:
-          node-version: 14.x
-      - name: Get next release branch
-        uses: ./.github/workflows/get-next-release-branch
-        with:
-          INPUT_VERSION: ${{ github.event.inputs.version }}
-      - name: Checkout next release branch
-        uses: actions/checkout@v3
-        with:
-          ref: ${{ env.RELEASE_BRANCH }}
-      - name: Install Dependencies
-        run: |
-          PACKAGE_VERSION=$(node -p "require('./package.json').last_cloud_version")
-          echo "PACKAGE_VERSION=${PACKAGE_VERSION}" >> $GITHUB_ENV
-          npm ci
-      - name: Build plugin
-        uses: ./.github/workflows/build-plugin
-        with:
-          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
-      - name: Upload zip file to GitHub actions artifact
-        uses: actions/upload-artifact@v2
-        with:
-          name: elementor-${{ env.PACKAGE_VERSION }}
-          path: ${{ github.workspace }}/elementor/**/*
-          if-no-files-found: error
-      - name: Create Git tag
-        uses : ./.github/workflows/create-git-tag
-        with:
-          MAINTAIN_USERNAME: ${{ secrets.MAINTAIN_USERNAME }}
-          MAINTAIN_EMAIL: ${{ secrets.MAINTAIN_EMAIL }}
-          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
-      - name: Generate changelog
-        uses : ./.github/workflows/generate-changelog
-        with:
-          TOKEN: ${{ secrets.MAINTAIN_TOKEN }}
-          REPOSITORY: ${{ github.repository }}
-          HEAD_BRANCH_NAME: v${{ env.PACKAGE_VERSION }}
-          BASE_TAG_NAME: v${{ needs.bump-version.outputs.prev_version }}
-      - name: Create GitHub release
-        uses: softprops/action-gh-release@v1
-        with:
-          tag_name: v${{ env.PACKAGE_VERSION }}
-          files: elementor-*.zip
-          prerelease: ${{ github.event.inputs.pre_release }}
-          body_path: temp-changelog.txt
+      - name: Publish Cloud
         env:
-          GITHUB_TOKEN: ${{ secrets.MAINTAIN_TOKEN }}
-      - name: Send slack message
-        uses: ./.github/workflows/slack-notify
-        with:
-          MESSAGE: "@here Elementor ${{ env.PACKAGE_VERSION }} has been released! Saddle up partners! :tada:"
-          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-          SLACK_CHANNEL: ${{ secrets.SLACK_CHANNELS }}
+          TOKEN: ${{ secrets.MAINTAIN_TOKEN }} # GitHub access token to access the remote repo.
+          REMOTE_PUBLISH_REPO: ${{ secrets.REMOTE_PUBLISH_REPO }} # Remote tests repo, e.g. `owner/repo-name`.
+          HEAD_REPO: ${{ github.event.pull_request.head.repo.full_name || github.repository }} # The HEAD repo that initiated the push/PR.
+        uses: peter-evans/repository-dispatch@d57191493a947e8da4f7bf3fbc389e5b20f21cdf
+        with:
+          token: ${{ env.TOKEN }}
```

Comment:
Why not use the secrets directly?

Reply:
This is code that I copy from `elementor-editor-dev/plugins/elementor/.github/workflows/elementor-tests.yml`.
I removed those envs end used the secrets directly.